src/jti_background.js

```diff
@@ -468,6 +481,16 @@ function templateDataToJSON (templates) {
     return formattedTemplates;
 }
 
+function domainDataToJSON (domains) {
+    var formattedDomains = [];
+
+    $.each(domains, function (key, domain) {
+        delete domain.id;
```

Comment:
I don't think we actually need to delete the id. Aren't we simply creating a list of the `domain.name`'s?

Reply:
:+1: